### ISSUE TITLE
On develop branch, when viewing metadata in Metadata tab, certain values are wrapped in quotation marks

### ISSUE THREAD

**Issue by jggautier** [IQSS/dataverse#8411]
This is on the develop branch (9b3604b). The metadata displays as intended on installations running 5.9 (e.g. Harvard Dataverse Repository and Demo Dataverse) and Scholars Portal Dataverse running 5.8.1 (forked).

**What steps does it take to reproduce the issue?**

* When does this issue occur?
On the develop branch, when users visit the Metadata tab of a dataset that certain metadata filled. So far I only see this when:
   - The field's displayFormat (in the metadatablock's tsv file) turns the text into a link
   - The field's parent field has a comma for its displayFormat. The Software field is the only field in the Citation metadata block that has this.

* Which page(s) does it occurs on?
On the metadata tab of a dataset page

* To whom does it occur (all users, curators, superusers)?
All users

* What did you expect to happen?
What users enter in the metadata fields is displayed on the metadata tab of a dataset page but without quotation marks.

**Which version of Dataverse are you using?**
The develop branch (as of the time of submitting this issue).

**Any related open or closed issues to this bug report?**
None that I could find.

**Screenshots:**
<img width="938" alt="Screen Shot 2022-02-09 at 9 32 31 AM" src="https://user-images.githubusercontent.com/18374574/153260236-6a5659d6-2d50-4b0d-94c3-07353c59d032.png">

**Comment by pdurbin** [comment]
Hmm, I'm not seeing the quotes:

![Screen Shot 2022-02-09 at 3 07 41 PM](https://user-images.githubusercontent.com/21006/153281760-0e81fd95-d53f-4761-937b-35f3c596e4fb.png)

**Comment by jggautier** [comment]
Thanks for checking. That's weird. I took a closer look at the displayFormat columns in [the citation.tsv file in the develop branch](https://github.com/IQSS/dataverse/blob/develop/scripts/api/data/metadatablocks/citation.tsv) and in [the citation.tsv file in my "tooltip-improvements" branch](https://github.com/IQSS/dataverse/blob/8127-citation-tooltip-improvements/scripts/api/data/metadatablocks/citation.tsv) and see that:
- the citation.tsv file in the develop branch has `<a href="#VALUE" target="_blank">#VALUE</a>` for the fields whose values turn into links, like the alternativeURL field, and `,` (a single comma) for the software field
- the citation.tsv file in the "tooltip-improvement" branch has those two values wrapped in quotes: `"<a href="#VALUE" target="_blank">#VALUE</a>"` and `","`

I'm going to close this GitHub issue. This seems like another problem with the way I'm editing or exporting the TSV file and another thing I'll have to keep an eye on as I edit and load these tsv files.

Interestingly, [the tsv file](https://github.com/IQSS/dataverse/blob/8127-citation-tooltip-improvements/scripts/api/data/metadatablocks/citation.tsv) I exported from Google Sheets and added to my "tooltip-improvements" branch displays nicely on the GitHub page. I wonder if that's because the top row of my citation.tsv file has 17 columns (5 non empty columns and 12 empty columns), which is the same number of columns as the row with the most number of columns (row three), which makes the tabular data a proper rectangle, while the top rows of the citation.tsv files in the master and develop branches have 16 columns (5 non empty columns and 11 empty columns).
